supplier_app/constants/email_notifications.py

```diff
@@ -55,4 +55,15 @@
             'btn_url': '{}{}'.format(settings.BRITESU_BASE_URL, SUPPLIER_HOME_URL),
         },
     },
+    'taxpayer_in_progress': {
+        'subject': _('Your taxpayer is in progress now'),
+        'body': {
+            'upper_text': _(
+                'You Taxpayer is now in progress to Approve\n'
+            ),
+            'lower_text': _('You dont need to do anything, we contact with you soon, Thanks!'),
```

Comment:
you don't need to do anything ELSE, we WILL contact you soon

Reply:
Done